### ISSUE TITLE
Voiceover: add to cart button read out incorrectly

### ISSUE THREAD

**Issue by ghost** [DivanteLtd/next#391]
Voiceover reads the Add to Cart option on listing pages (list view) as 'A D D to cart' not 'Add to cart'

**Comment by Qrzy** [comment]
Reported to SFUI team: https://github.com/DivanteLtd/storefront-ui/issues/1070

**Comment by Qrzy** [comment]
It turns out that properly set screen reader does not have such issue.
